src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -122,23 +122,42 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
         _UpdateOverrideSystem();
 
-        if (const auto& content{ Content() })
+        if (const auto& child{ GetTemplateChild(L"Expander") })
+        {
+            if (const auto& expander{ child.try_as<Microsoft::UI::Xaml::Controls::Expander>() })
+            {
+                // apply header text as name (automation property)
+                if (const auto& header{ Header() })
+                {
+                    if (const auto headerText{ header.try_as<hstring>() })
+                    {
+                        Automation::AutomationProperties::SetName(expander, *headerText);
+                    }
+                }
+
+                // apply help text as tooltip and full description (automation property)
+                if (const auto& helpText{ HelpText() }; !helpText.empty())

```

Comment:
`if (const auto& helpText{ HelpText() })` does not work for some reason

Reply:
this is great then